src/workloads/tpch/normalized/10/Q9_normalized_10.yml

```diff
@@ -21,5 +21,5 @@ Actors:
 #   Database: tpch
 #   Phases:
 #   - LoadConfig:
-#       Path: ../../../../phases/tpch/normalized/Q19.yml
```

Comment:
Looks like a typo from the original workload, probably should be Q9 instead of Q19.